trainer/derrian_backend/sd_scripts/tools/cache_text_encoder_outputs.py

```
@@ -178,7 +179,52 @@ def cache_to_disk(args: argparse.Namespace) -> None:
+        n_workers = min(args.max_data_loader_n_workers, os.cpu_count())  # cpu_count or max_data_loader_n_workers
+
+        train_dataloader = torch.utils.data.DataLoader(
+            train_dataset_group,
+            batch_size=1,
+            shuffle=True,
+            collate_fn=collator,
+            num_workers=n_workers,
+            persistent_workers=args.persistent_data_loader_workers,
+        )
+
+        # acceleratorを使ってモデルを準備する：マルチGPUで使えるようになるはず
```

Ruff (0.15.12)

[warning] Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)